Makefile

```diff
@@ -415,32 +403,121 @@ release-binary: $(RELEASE_DIR) versions.mk
 		-e CGO_ENABLED=0 \
 		-e GOOS=$(GOOS) \
 		-e GOARCH=$(GOARCH) \
+		-v source=gocache,target=/root/.cache/go-build \
 		-v "$$(pwd):/workspace$(DOCKER_VOL_OPTS)" \
 		-w /workspace \
 		golang:$(GOLANG_VERSION) \
-		go build -a -ldflags '$(LDFLAGS) -extldflags "-static"' \
+		go build -ldflags '$(LDFLAGS) -extldflags "-static"' \
 		-o $(RELEASE_DIR)/$(notdir $(RELEASE_BINARY))-$(GOOS)-$(GOARCH) $(RELEASE_BINARY)
 
 .PHONY: release-staging
-release-staging: ## Builds and push container images to the staging bucket.
-	REGISTRY=$(STAGING_REGISTRY) $(MAKE) docker-build-all docker-push-all release-alias-tag
+release-staging: ## Builds and push container images and manifests to the staging bucket.
+	$(MAKE) docker-build-all
+	$(MAKE) docker-push-all
+	$(MAKE) release-alias-tag
+	$(MAKE) staging-manifests
+	$(MAKE) upload-staging-artifacts
 
-RELEASE_ALIAS_TAG=$(PULL_BASE_REF)
+.PHONY: upload-staging-artifacts
+upload-staging-artifacts: ## Upload release artifacts to the staging bucket
+	gsutil cp $(RELEASE_DIR)/* gs://$(BUCKET)/components/$(RELEASE_ALIAS_TAG)
+
+.PHONY: create-gh-release
+create-gh-release:$(GH) ## Create release on Github
```

Comment:
I observe this error on Mac:

```
 tar -xvf share/gh/gh.tar.gz gh_1.2.0_macOS_amd64/bin/gh  --strip-components 1 --directory bin
x gh_1.2.0_macOS_amd64/bin/gh
tar: --strip-components: Not found in archive

```

Reply:
Switched to gtar, and added a check to make sure GNU Tar is installed.

Reply:
Thought I pulled the updated one, will rebase and try again.

Reply:
Yep works.

Reply:
> Thought I pulled the updated one, will rebase and try again.

Had updated after you commented. :)